### PR TITLE
idba: add hybrid and tran variants

### PR DESCRIPTION
**Pull request by bernt-matthias**
supersedes also https://github.com/galaxyproject/tools-iuc/pull/2492

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/idba_ud/idba_hybrid.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <param argument="--min_region" type="integer" value="300" label="Minimum size of region in reference genome"/>
+        <param argument="--max_gap" type="integer" value="3" label="Maximum gap in reference"/>
+        <expand macro="min_pairs_filter"/>
+        <expand macro="other_options"/>
+    </inputs>
+    <outputs>
+        <data name="output" from_work_dir="out/scaffold.fa" format="fasta"/>
+    </outputs>
+    <tests>
+        <test>
```
Comment (bgruening) [inline tools/idba_ud/idba_hybrid.xml]: a test for the loc file case would be nice but not strictly needed

Comment (bernt-matthias) [reply tools/idba_ud/idba_hybrid.xml]: you are absolutely right. and in this case it was needed.

**Comment by bernt-matthias** [comment]
I adapted the shed.yml. Please double check.

**Comment by bernt-matthias** [comment]
sure. give me a bit.

**Comment by bernt-matthias** [comment]
there you go. you are absolutely right tests are important -- as one of our admins put it: "uebergangsloesungen sind untergangsloesungen"

note that the `assert_command` parts currently seem to have no effect: https://github.com/galaxyproject/galaxy/pull/7730 (maybe I can create some attention this way).

**Comment by bgruening** [comment]
Merci!

**Pull request merged**